src/ParallelAlgorithms/Amr/Events/Events.hpp

```diff
@@ -4,7 +4,5 @@
 #pragma once
 
 /// \ingroup AmrGroup
-namespace amr {
 /// \brief %Eventss for adaptive mesh refinement
-namespace Events {}
-}  // namespace amr
+namespace amr::Events {}  // namespace amr::Events
```

Comment:
Move reformatting to the commit where it was added.  Also remove the same-line closing comment.